news/2023/2023-03-03-mappers-choice-awards.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Mapper's Choice Awards / A Year In Mapping 2022"
+date: 2023-03-03 00:00:00 +0000
```

Comment:
osu!monthly is scheduled in the next hour or so, so i think putting this a bit into the day is more reasonable

```suggestion
date: 2023-03-03 09:00:00 +0000
```